### PR TITLE
Add Gabriel Fukushima as a maintainer

### PR DESCRIPTION
**Pull request by jframe**
Signed-off-by: Jason Frame <jason.frame@consensys.net>

## PR description
Proposing @gfukushima as a new Besu maintainer. They have made over 5 significant [commits](https://github.com/hyperledger/besu/pulls?q=is%3Apr+author%3Agfukushima+is%3Aclosed) that have improved the quality of Besu and/or added new features.

Voting ends 2 weeks from the publication of this PR or once more than 50% of the current maintainers approve.

For more information on this process see the Becoming a Maintainer section in the MAINTAINERS.md file.

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if
    [updates are required](https://wiki.hyperledger.org/display/BESU/Documentation).

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -6,32 +6,33 @@
...
+| Ameziane Hamlat     | ahamlat             | ahamlat            |
+| Adrian Sutton       | ajsutton            | ajsutton           |
+| Antony Denyer       | antonydenyer        | antonydenyer       |
+| Antoine Toulme      | atoulme             | atoulme            |
+| Byron Gravenorst    | bgravenorst         | bgravenorst        |
+| Daniel Lehrner      | daniellehrner       | daniellehrner      |
+| Diego López León    | diega               | diega              |
+| Fabio Di Fabio      | fab-10              | fab-10             |
+| Gary Schulte        | garyschulte         | GarySchulte        | 
 | Gabriel Trintinalia | gabriel-trintinalia | gabrieltrintinalia |
```
Comment (siladu) [inline MAINTAINERS.md:20]: > alphabetically by github

gets ppl every time :)

**Review by siladu — APPROVED** [review]
LGTM

**Review by macfarla — APPROVED** [review]
LGTM

**Review by lucassaldanha — APPROVED** [review]

**Review by usmansaleem — APPROVED** [review]
LGTM

**Review by daniellehrner — APPROVED** [review]

**Review by Gabriel-Trintinalia — APPROVED** [review]
LGTM

**Review by garyschulte — APPROVED** [review]
:+1:

**Comment by ahamlat** [comment]
👍

**Review by mark-terry — APPROVED** [review]
LGTM.

**Review by fab-10 — APPROVED** [review]

**Review by diega — APPROVED** [review]

**Review by jflo — APPROVED** [review]

**Comment by garyschulte** [comment]
seems to me this should be set to auto-merge.  With the original author, I count 13 of 24

**Comment by garyschulte** [comment]
ah, @ahamlat - you only commented.  You didn't "approve"

**Review by ajsutton — APPROVED** [review]

**Pull request merged**